kiosk/src/Components/QrScanner.tsx

```diff
@@ -11,6 +12,7 @@ export const play = async (kiosk: Kiosk, kioskId: string, html5QrCode: Html5Qrco
         const shareId = /\/([^\/]+)\/?$/.exec(decodedText)?.[1];
         try {
             await addGameToKioskAsync(kioskId, shareId);
+            tickEvent("kiosk.gameQrScanned.success");
             await html5QrCode.stop();
             kiosk.navigate(KioskState.QrSuccess);
         } catch (error) {
```

Comment:
Add a tick event if the gameQRScan fails?

Reply:
Unfortunately, this tick would be too noisy. I'm not sure why the api works this way, but the scanning is indicated as failing if the reader doesn't see a QR code, and this will happen before anyone even attempts to scan a QR code.